### ISSUE TITLE
How to add and store  title/name/alt for image / multiple image in voyager?

### ISSUE THREAD

**Issue by yevgenatlanov** [thedevdojo/voyager#3009]
- Laravel Version: 5.5
- Voyager Version: 1.0.*
- PHP Version: 7.0
- Database Driver & Version: mysql 5.7

### Description:
How can i add and store title/name or alt for image / multiple images in voyager media manager or in the simple default bread views? Any ideas?

**Comment by emptynick** [comment]
Is this a duplicate of your own issue https://github.com/the-control-group/voyager/issues/2968 ?

**Comment by yevgenatlanov** [comment]
@emptynick yep, i've changed the context of the question. Looking for advice.

**Comment by emptynick** [comment]
You can change the Title and the body of your **issue**.
Also please ask question in our Slack group.
We try to separate issues and questions.

**Comment by bnku** [comment]
I released the hook about this case:
https://github.com/bnku/extended-bread-form-fields
